UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.UnitTests/Filters/BespokeExchangeSetAuthorizationFilterAttributeTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿using FakeItEasy;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Primitives;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using UKHO.ExchangeSetService.API.Filters;
+
+namespace UKHO.ExchangeSetService.API.UnitTests.Filters
+{
+    [TestFixture]
+    public class BespokeExchangeSetAuthorizationFilterAttributeTests
+    {
+        private IConfiguration fakeConfiguration;
+        private BespokeExchangeSetAuthorizationFilterAttribute bespokeFilterAttribute;
+        private ActionExecutingContext actionExecutingContext;
+        private ActionExecutedContext actionExecutedContext;
+        private const string TokenAudience = "aud";
+        private const string IsUnencrypted = "IsUnencrypted";
+        private const string ESSAzureADConfigurationClientId = "ESSAzureADConfiguration:ClientId";
```

Comment:
Can you please move these magic strings (line 27 and line 28) into a resource file and reference from there, this will support us maintaining.

Reply:
Resolved

Reply:
Magic string at line 27 is removed.